cli/bin/grain.js

```diff
@@ -189,28 +175,27 @@ program
   .command("compile-and-run <file>", { isDefault: true, hidden: true })
   // `--version` should only be available on the default command
   .version(pkgJson.version, "-v, --version", "output the current version")
-  .addOption(new commander.Option("-p, --print-output").hideHelp())
   .forwardOption("-o <filename>", "output filename")
   .action(function (file, options, program) {
     exec.grainc(file, options, program);
     if (options.o) {
-      run(options.o, options);
+      exec.grainrun(options.o, options, program);
     } else {
-      run(file.replace(/\.gr$/, ".gr.wasm"), options);
+      exec.grainrun(file.replace(/\.gr$/, ".gr.wasm"), options, program);
     }
   });
 
 program
   .command("compile <file>")
   .description("compile a grain program into wasm")
   .forwardOption("-o <filename>", "output filename")
+  .forwardOption("--no-link", "disable static linking")
   .action(exec.grainc);
 
 program
   .command("run <file>")
-  .description("run a wasm file with grain's javascript runner")
-  .addOption(new commander.Option("-p, --print-output").hideHelp())
-  .action(run);
+  .description("run a wasm file via Node.js")
```

Comment:
```suggestion
  .description("run a wasm file via grain's WASI runner")
```
I think we should say this because eventually this will be wamr.